src/templates/upgrade.tsx

```diff
@@ -185,130 +249,125 @@ const components = {
   QuizWidget,
 }
 
-const Title = styled.h1`
-  font-size: 2.5rem;
-  font-weight: 700;
-  margin-top: 0rem;
-`
-
-const SummaryPoint = styled.li`
-  font-size: 1rem;
-  color: ${(props) => props.theme.colors.text300};
-  margin-bottom: 0rem;
-  line-height: auto;
-`
-
-const SummaryBox = styled.div`
-  /* border: 1px solid ${(props) => props.theme.colors.border};
-  padding: 1.5rem;
-  padding-bottom: 0rem;
-  border-radius: 4px; */
-`
-
-const DesktopBreadcrumbs = styled(Breadcrumbs)`
-  margin-top: 0.5rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-const MobileBreadcrumbs = styled(Breadcrumbs)`
-  margin-top: 0.5rem;
-  @media (min-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const StyledButtonDropdown = styled(ButtonDropdown)`
-  margin-bottom: 2rem;
-  display: flex;
-  justify-content: flex-end;
-  text-align: center;
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    align-self: flex-end;
-  }
-`
+const Title = (props: ChildOnlyProp) => (
+  <Heading
+    as="h1"
+    fontSize="2.5rem"
+    fontWeight="bold"
+    lineHeight={1.4}
+    mt={0}
+    {...props}
+  />
+)
+
+const SummaryPoint = (props: ChildOnlyProp) => (
+  <ListItem color="text300" mb={0} {...props} />
+)
+
+const DesktopBreadcrumbs = chakra(Breadcrumbs, {
+  baseStyle: {
+    mt: 2,
+    display: { base: "none", lg: "block" },
+  },
+})
+
+const MobileBreadcrumbs = chakra(Breadcrumbs, {
+  baseStyle: {
+    mt: 2,
+    display: { lg: "none" },
+  },
+})
```

Comment:
Good point. Not sure why we had it like that. But anyway, lets refactor it a bit.

We can remove these two since we are always going to display the Breadcrumbs to the user. We can just use the Breadcrumbs component and apply the necessary styles on it.

```tsx
// remove these
<DesktopBreadcrumbs slug={slug} startDepth={1} />
<MobileBreadcrumbs slug={slug} startDepth={1} />

// just use
<Breadcrumbs slug={slug} startDepth={1} mt={2} />
```